controller_joint_names: ['', 'rear_wheel_left_joint', 'rear_wheel_right_joint', 'front_wheel_left_steering_joint', 'front_wheel_left_joint', 'front_wheel_right_steering_joint', 'front_wheel_right_joint', ]
662_bot :
# default controller
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50

  # Controller 1
  rear_wheel_left_joint_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "rear_wheel_left_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0} 

  # Controller 2s
  rear_wheel_right_joint_controller:
    type: velocity_controllers/JointVelocityController 
    joint: "rear_wheel_right_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0}
  
  #Controller 3
  front_wheel_left_steering_joint_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_left_steering_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0}
  
  #Controller 4
  front_wheel_left_joint_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_left_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0}
  
  #Controller 5
  front_wheel_right_steering_joint_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_right_steering_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0}

  #Controller 6
  front_wheel_right_joint_controller:
    type: effort_controllers/JointPositionController
    joint: "front_wheel_right_joint"
    pid : {p: 100.0, i: 0.01, d: 10.0}